src/math.jl

```diff
@@ -715,6 +715,9 @@ Base.nextfloat(a::Measurement, n::Integer) = result(nextfloat(a.val, n), 1, a)
 
 Base.maxintfloat(::Type{Measurement{T}}) where {T<:AbstractFloat} = maxintfloat(T)
 
+Base.floatmin(::Type{Measurement{T}}) where {T<:AbstractFloat} = floatmin(T)
+Base.floatmax(::Type{Measurement{T}}) where {T<:AbstractFloat} = floatmax(T)
```

Comment:
I'm undecided whether this should be 
```suggestion
Base.floatmin(::Type{Measurement{T}}) where {T<:AbstractFloat} = floatmin(T) ± zero(T)
Base.floatmax(::Type{Measurement{T}}) where {T<:AbstractFloat} = floatmax(T) ± zero(T)
```
instead.  I don't have a good use in mind for this.

Reply:
The latter seems preferable, but I assumed there was some rationale for the preexisting `Base.typemax` definition directly below the methods I added, so I tried to conform to the same structure.

Reply:
> I assumed there was some rationale for the preexisting Base.typemax definition directly below the methods I added

Don't assume I always make rational choices :stuck_out_tongue:

In #64 we struggled to track down a bug which turned out to be due to the the fact `nextfloat` was returning a non-`Measurement` object (fixed in #65).  For this reason, now I tend to think that these functions shouldn't return an object without uncertainty, as we may loose the information about the type, which can open all sort of issues.